js/main.js

```diff
@@ -0,0 +1,27 @@
+var $img = document.querySelector('.journal-image');
+var $title = document.querySelector('#title');
+var $photoURL = document.querySelector('#img-url');
+var $notes = document.querySelector('#notes');
+var $form = document.querySelector('.entry-form');
+
+function updatePhoto(event) {
+  $img.setAttribute('src', $photoURL.value);
+}
+
+function saveEntry(event) {
+  event.preventDefault();
+  var newEntry = {};
+  newEntry.title = $title.value;
+  newEntry.photoURL = $photoURL.value;
+  newEntry.notes = $notes.value;
+  newEntry.EntryID = data.nextEntryId;
```

Comment:
Property names should not begin with a capital letter, please rename this `entryId`

Reply:
Thanks for catching that Cody, I definitely had this as `newEntry.NextEntryID `and caught it because it didn't make sense so I deleted the next but missed on the E and D.

![image](https://user-images.githubusercontent.com/109567499/189770461-ba6680b6-5c22-427e-954d-de3c717aa6af.png)